### PR TITLE
bugfix missing years in f43_wat_avail

### PR DESCRIPTION
**Pull request by flohump**
## :bird: Purpose of this PR :bird:

- **43_water_availability** added missing years after 2100 in "f43_wat_avail" to avoid infeasibilities in coupled runs with less_ts timesteps
- **scripts/start** cleanup of old start scripts
- Run time and results for default runs are identical to the current develop
- With this bugfix runs with `less_ts` work again.
- FSEC run is feasible

## :wrench: Checklist for PR creator :wrench:

- [x] Labeling pull request correctly [from the label list](https://github.com/magpiemodel/magpie/labels).

* Low risk : Simple bugfixes (missing files, updated documentation, typos) or Start/output scripts
* Medium risk : New realization / Changes to existing realization / Other changes which don't modify default.cfg
* High risk : New input files (if cfg$input is changed in default.cfg) / Modification to core model (eg. changes in equations, calculations, introduction of new sets etc.) / Other changes in default.cfg

- [x] Providing additional information based on PR label

* Low risk : No new model run needed.
* Medium risk : Default run based on the current version of the fork from which PR is made
* High risk
  * Default run from the current develop branch: PR422_currentDevelop
  * Default run based on the current version of the fork from which PR is made: PR422_thisPR

Resources_Land_Cover_Cropland-74
Productivity_Landuse_Intensity_Indicator_Tau-63
Emissions_CO2_Land_Land_use_Change-55

- [x] :chart_with_downwards_trend: Performance loss/gain from current default behavior :chart_with_upwards_trend:
  * Current develop branch default : 0.426 h
  * This PR's default :  0.429 h

- [x] Added changes to `CHANGELOG.md`
- [x] Compilation check (model starts without compilation errors - use `gams main.gms action=c` in model folder for testing).
- [x] No hard coded numbers and cluster/country/region names.
- [x] The new code doesn't contain declared but unused parameters or variables.
- [x] Where relevant, In-code comments added including documentation comments.
- [x] Made sure that documentation created with [`goxygen`](https://github.com/pik-piam/goxygen) is okay (use `goxygen::goxygen()` for testing).
- [x] Changes to [`magpie4`](https://github.com/pik-piam/magpie4) R library for post processing of model output (ideally backward compatible).
- [x] Self-review of my own code.
- [x] In case of updated cellular input tgz file in default.cfg: scenario_config.csv has been updated accordingly (rcp1p9, rcp2p6 etc)
- [x] For high risk runs: validation of major model indicators - Land-use, emissions, food prices, Tau.  %Delete this line in case it is not a high risk run%

## :warning: Additional notes or warnings :warning:
NA

## :rotating_light: Checklist for RSE reviewer :rotating_light:

- [ ] PR is labeled correctly.
- [ ] `CHANGELOG` is updated correctly
- [ ] No hard coded numbers and cluster/country/region names.
- [ ] No unnecessary increase in module interfaces
- [ ] All required updates in interfaces (if any) have been properly adressed in the module contracts
- [ ] In-code comments and documentation comments are satisfactory.
- [ ] model behavior/performance is satisfactory.
- [ ] Requested changes (if any) were applied correctly

## :rotating_light: Checklist for MAgPIE reviewer :rotating_light:

- [ ] PR is labeled correctly.
- [ ] `CHANGELOG` is updated correctly
- [ ] No hard coded numbers and cluster/country/region names.
- [ ] Changes to the model are scientifically sound
- [ ] In-code comments and documentation comments are satisfactory.
- [ ] model behavior/performance is satisfactory.
- [ ] Requested changes (if any) were applied correctly

### CODE REVIEW

**Review by abhimishr — APPROVED** [review]

**Review by abhimishr — COMMENTED** [review]
sorry for approving draft PR. Looked straight forward based on mattermost discussion.

File: scripts/start/projects/paper_ClimNat.R [hunk 1]
```diff
@@ -17,6 +17,7 @@
 library(gms)
 library(lucode2)
 library(magclass)
+library(magpie4)
```
Comment (mikapfl) [inline scripts/start/projects/paper_ClimNat.R:20]: Why do you need to import `magpie4` here?

**Review by vartika271987 — APPROVED** [review]

File: scripts/start/projects/paper_ClimNat.R (hunk 1 above)
Comment (flohump) [reply scripts/start/projects/paper_ClimNat.R]: It's not needed. I removed it.

**Review by emolinab — APPROVED** [review]

File: scripts/start/projects/paper_ClimNat.R [hunk 2]
```diff
@@ -25,82 +25,90 @@ source("scripts/start_functions.R")
...
       cfg$gms$c35_forest_damage_end <- "by2030"
       cfg$gms$s44_target_price <- 0
+      cfg$gms$s44_bii_target <- 0
       cfg$gms$c35_protect_scenario <- "WDPA"
       cfg$gms$c30_snv_target <- "none"
       cfg$gms$s30_snv_shr <- 0
-      cfg$gms$c56_emis_policy <- "redd+_nosoil"
-    } else if (pol == "Nature") {
-      cfg <- setScenario(cfg,c(ssp,"NDC","rcp7p0"))
+      #cfg$gms$c56_emis_policy <- "redd+_nosoil"
```
Comment (pfuehrlich-pik) [inline scripts/start/projects/paper_ClimNat.R:76]: I'd suggest a comment explaining why this commented out code should stay, or just delete the line (git allows to recover it anyway).

File: scripts/start/projects/paper_scp.R [hunk 3]
```diff
@@ -18,55 +18,37 @@ library(gms)
 library(lucode2)
 library(magclass)
 
-# Load start_run(cfg) function which is needed to start MAgPIE runs
+# source functions needed to start MAgPIE runs
```
Comment (pfuehrlich-pik) [inline scripts/start/projects/paper_scp.R:21]:
```suggestion
```
I'd say the code is self explanatory here.

File: scripts/start/projects/paper_scp.R [hunk 4]
```diff
@@ -18,55 +18,37 @@ library(gms)
...
 source("scripts/start_functions.R")
 
-#start MAgPIE run
+# source default config file
```
Comment (pfuehrlich-pik) [inline scripts/start/projects/paper_scp.R:24]:
```suggestion
```
The code is definitely self explanatory here, it's even using the same words as the comment :smile:

File: scripts/start/projects/paper_scp.R [hunk 5]
```diff
@@ -18,55 +18,37 @@ library(gms)
...
-      start_run(cfg,codeCheck=FALSE)
-    }
-  }
+# replacement of ruminant meat with MP
+cfg$gms$kfo_rd <- "livst_rum"
+
+# Start MAgPIE runs with varying substitution targets of ruminant meat with MP by 2050, based on protein/cap/day basis
+for (MP in c(0,20,50,80)) {
+  cfg$title <- paste(prefix,paste0("SSP2-Ref-MP",MP),sep="_")
+  if (MP == 0) scp_scen <- "constant" else if (MP == 20) scp_scen <- "sigmoid_80pc_20_50" else if (MP == 50) scp_scen <- "sigmoid_50pc_20_50" else if (MP == 80) scp_scen <- "sigmoid_20pc_20_50"
```
Comment (pfuehrlich-pik) [inline scripts/start/projects/paper_scp.R:51]: I find this line hard to read, I'd appreciate a couple line breaks.

**Review by pfuehrlich-pik — COMMENTED** [review]
Just some minor comments. There are a bunch of commented out lines of code that I don't find helpful. That might be because I'm still a magpie newcomer, but I also have the feeling that other (newcomer?) team members may also be irritated. I'd suggest adding comments to the commented out code explaining when the code should be commented in again, or why it is there. Alternatively just delete the commented code and rely on git to get it back if needed.

**Comment by flohump** [comment]
> Just some minor comments. There are a bunch of commented out lines of code that I don't find helpful. That might be because I'm still a magpie newcomer, but I also have the feeling that other (newcomer?) team members may also be irritated. I'd suggest adding comments to the commented out code explaining when the code should be commented in again, or why it is there. Alternatively just delete the commented code and rely on git to get it back if needed.

I removed the commented out lines of code in the start scripts.

**Review by pfuehrlich-pik — APPROVED** [review]
Looks good!

**Pull request merged**